webservices/tasks/download.py

```diff
@@ -190,9 +148,17 @@ def make_filter(key, value, description):
         lines.append(description.strip())
     return '\n'.join(lines)
 
-def make_bundle(resource, query):
+def wc(path):
```

Comment:
This might be nit-picky, but `wc` as a name is a bit unclear when just reading through the code (though my assumption is that it's in reference to the `wc` UNIX utility after having walked through testing the approach of this with you before).